gnd/src/main/res/layout/basemap_selector_frag.xml

```diff
@@ -0,0 +1,32 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.google.android.gnd.ui.basemapselector.BasemapSelectorViewModel" />
+    </data>
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <RelativeLayout
+            android:id="@+id/map_layout"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_gravity="top"
+            android:gravity="center"
+            app:layout_behavior="com.google.android.gnd.ui.home.mapcontainer.MapLayoutBehavior">
+
+            <!-- Featureholder for map implementation -->
```

Comment:
s/featureholder/placeholder/?

Reply:
Yes, done!

Actually, the map is already rendered, so I just removed the comment.